src/checkers/inference/InferenceVisitor.java

```
@@ -853,4 +854,33 @@ protected InferenceValidator createTypeValidator() {
     protected void checkConstructorResult(
             AnnotatedExecutableType constructorType, ExecutableElement constructorElement) {}
     
+    @Override
+    public Void visitUnary(UnaryTree node, Void p) {
+        if (!infer) {
+            return super.visitUnary(node, p);
+        }
+
+        switch (node.getKind()) {
```

Add `// fall through ` to suppress error-prone warning